frontend/views/components/Lists/index.js

```diff
@@ -1,2 +1 @@
-export { default as List } from './List.vue'
 export { default as ListItem } from './ListItem.vue'
```

Comment:
If all this file does is import ListItem, then the folder Lists should be deleted itself, and the file ListItems moved up a directory